src/vm/memory/memory.zig

```diff
@@ -207,6 +207,69 @@ pub const Memory = struct {
     }
 };
 
+// Utility function to help set up memory for tests
+//
+// # Arguments
+// - `memory` - memory to be set
+// - `vals` - complile time structure with heterogenous types
+fn memoryInner(memory: *Memory, comptime vals: anytype) !void {
+    inline for (vals) |row| {
+        const firstCol = row[0];
+        const relo = Relocatable.new(firstCol[0], firstCol[1]);
+        // Check number of inputs in row
+        if (row[1].len == 1) {
+            const nextCol = row[1];
+            const maybeFromFelt = MaybeRelocatable{ .felt = Felt252.fromInteger(nextCol[0]) };
+            try memory.set(relo, maybeFromFelt);
+        } else {
+            const val = row[1];
```

Comment:
As this approach is used both here (`const val = row[1];`) and also in the main if (`const nextCol = row[1];`) perhaps we could declare the constant in a general outside of the if, before no?